### ISSUE TITLE
Windows wheel binaries for python 11, 12

### ISSUE THREAD

**Issue by CSharpDummy** [xyb/rocksdb3#10]
Would be nice to have those binaries. Tried to pip install on Windows with Python 3.12 and it wouldnt work.

**Comment by xyb** [comment]
@CSharpDummy Yes, it's essential to schedule a new release. I'll make sure to allocate some time to complete it soon.
